src/protocol.cpp

```diff
@@ -82,6 +82,8 @@ const char *CLSIG="clsig";
 const char *ISLOCK="islock";
 const char *ISDLOCK="isdlock";
 const char *MNAUTH="mnauth";
+const char *GETQUORUMROTATIONINFO="getquorumrotationinfo";
+const char *QUORUMROTATIONINFO="getquorumrotationinfo";
```

Comment:
Looks like this is a copy/paste error?
```suggestion
const char *QUORUMROTATIONINFO="quorumrotationinfo";
```

Reply:
@thephez Yes..I will fix that